velox/dwio/dwrf/reader/DwrfReaderShared.h

```diff
@@ -43,6 +43,7 @@ class DwrfRowReaderShared : public StrideIndexProvider,
   uint64_t strideIndex_;
   std::shared_ptr<StripeDictionaryCache> stripeDictionaryCache_;
   dwio::common::RowReaderOptions options_;
+  FileFormat fileFormat_;
```

Comment:
Do we really need this member variable?

Reply:
I thought it would be useful when RowReader needs to deal with the difference between DWRF/ORC, but my change has not touch that part. Removed.